### ISSUE TITLE
I've added jpeg support

### ISSUE THREAD

**Issue by Zackio** [ryanpcmcquen/import-external-attachments#4]
I've added jpeg support but because of the way the plugins code is done and my lack of time to rewrite those parts I've done it at the expense of jpe - I don't think that's going to come up as much as jpeg does.  But because I've removed jpe I've not done a pull request. If anyone needs jpeg you can use my folk.

**Comment by ryanpcmcquen** [comment]
I would prefer `jpeg` over `jpe` and would merge that PR.

**Comment by ryanpcmcquen** [comment]
Are you sure `jpeg` is not currently working? A sandbox test of the regex would suggest otherwise ...

https://repl.it/C5iK

**Comment by Zackio** [comment]
That regex works sure but that wasn't where I found the problem. It was in the is_external_file function and the fact it was getting the extension by getting the last 4 characters (it includes the dot), but because jpeg is a longer ext it didn't work.

It also needed to be added in external_image_get_img_tags but I found that having the jpe was breaking it so I removed that for quickness. I'm sure a regex rewrite would resolve that but regex is not my strong suit and I can't see that jpe is important.
